### PR TITLE
add contact module and responsive media queries

### PR DESCRIPTION
**Pull request by MonkeyKode**
Contact module added forms and buttons 
Already responsive.

### CODE REVIEW

File: contacto.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (willbackslash) [inline contacto.html]: Avoid empty lines between tabs

File: contacto.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                        <label for="mensaje">Mensaje</label>
+                        <textarea id="mensaje"></textarea>
+                    </div>
+                    <div class="campo enviar">
+                        <input type="submit" value="Enviar" class="btn btn-primario">
+                    </div>
+                </form>
+            </div>
+        </div>
+
```
Comment (willbackslash) [inline contacto.html]: same shit

File: css/style.css [hunk 3]
```diff
@@ -219,4 +244,37 @@ img{
...
+.formulario-contacto{
+    background-color: #ffffff;
+    padding: 5rem;
+}
+
+@media (min-width: 768px) {
+    .formulario-contacto{
+        margin-top: -10rem;
+    }
+
```
Comment (willbackslash) [inline css/style.css]: no empty line

File: cursos.html [hunk 4]
```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (willbackslash) [inline cursos.html]: same, no empty lines

**Review by willbackslash — CHANGES_REQUESTED** [review]

File: contacto.html (hunk 1 above)
Comment (MonkeyKode) [reply contacto.html]: tabs or tags?

**Review by willbackslash — APPROVED** [review]
LGTM :shipit:

**Pull request merged**